src/api-service/__app__/onefuzzlib/azure/vmss.py

```diff
@@ -177,6 +179,8 @@ def reimage_vmss_nodes(name: UUID, vm_ids: List[UUID]) -> Optional[Error]:
 
 def delete_vmss_nodes(name: UUID, vm_ids: List[UUID]) -> Optional[Error]:
```

Comment:
I wonder if these methods should instead take `Set[UUID]`.

Reply:
Eh, it pushes the set requirement up the stack, but I don't think it has any negative impact.

Done

Reply:
Yeah, I think either one is fine. A list is nice ergonomically, because they're sort of the lingua franca container in Python, while requiring a set is more precise, and makes it more likely that callers avoid building a collection with duplicates in the first place.